src/clims/services/application.py

```diff
@@ -4,7 +4,8 @@
 from clims.services.substance import SubstanceService
 from clims.services.container import ContainerService
 from clims.services.project import ProjectService
-from sentry.plugins import plugins
+from clims.services.workflow import WorkflowService
```

Comment:
As far as I understand, since workflow is something configurable, shouldn't it be included by plugins? Perhaps we could have a whiteboard discussion about how the configuration of workflow is thought to happen?

Reply:
Yeah, no doubt, let's do that. This is a part of my major branch that I'm sending in now. The plugins do configure the workflows there.